boards/st/nucleo_l452re/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: nucleo_l452re
+  vendor: st
+  socs:
+    - name: stm32l452xx
```

Comment:
Missing variant to indicate p variant is available

Reply:
Done